SparkPost/2web2ui
# [AC-1775] Removed the tfa_required condition for starter plans

[AC-1775] Removed the tfa_required condition for starter plans

### What Changed
 - Removed the hardcoded condition for starter plans
 - tfa_required toggle accurately represents the api value


**Another update**
 - Update the language of the plan change page to say Your new plan no longer allows for single sign-on.  instead of 'Your new plan no longer allows for single sign-on and account-wide requirement of two-factor authentication.'

### How To Test
 - Added a cypress test to check this change.
 - Trying to figure out an account on staging where this can be tested.

### To Do


[AC-1775]: https://sparkpost.atlassian.net/browse/AC-1775
